src/components/Pages/ManageWiki/Tabs/ManageFeatures.vue

```diff
@@ -1,13 +1,22 @@
 <template>
   <v-content>
-      Feature Settings
+    <FederatedProperties :wikiId="this.wikiId" />
```

Comment:
I'm amazed that it works with `this.`. Pretty sure it shouldn't be there, though! :D
```suggestion
    <FederatedProperties :wikiId="wikiId" />
```

Reply:
done!